### PR TITLE
Skip amenities list check

### PR DESCRIPTION
**Pull request by kocio-pl**
A bit of code cleaning. Basic testing has been done to check that the code compiles and amenities are still here.

### CODE REVIEW

**Comment by math1985** [comment]
Does this have performance implications? That was the reason the WHERE statements where there in the first place.

**Comment by kocio-pl** [comment]
It's just as big as shops list, so I guess they should unified this way (skipping) or another (providing full list).

**Comment by math1985** [comment]
I believe the important factor is not the length of the list, but the total amount of amenities (versus the amount we are selecting) in a given bounding box.

File: project.mml [hunk 1]
```diff
@@ -1442,14 +1442,7 @@ Layer:
...
-            OR amenity IN ('shelter', 'atm', 'bank', 'bar', 'bicycle_rental', 'bus_station', 'cafe',
-                           'car_rental', 'car_wash', 'cinema', 'clinic', 'community_centre',
-                           'fire_station', 'fountain', 'fuel', 'hospital', 'ice_cream', 'embassy', 'library', 'courthouse',
-                           'townhall', 'parking', 'bicycle_parking', 'motorcycle_parking', 'pharmacy', 'doctors',
-                           'dentist', 'place_of_worship', 'police', 'post_box', 'post_office', 'pub', 'biergarten',
-                           'recycling', 'restaurant', 'food_court', 'fast_food', 'telephone', 'taxi',
-                           'theatre', 'toilets', 'drinking_water', 'prison', 'hunting_stand', 'nightclub', 'veterinary',
-                           'social_facility', 'charging_station', 'arts_centre', 'ferry_terminal')
+            OR amenity IS NOT NULL -- skip checking a huge list and use a null check
             OR shop IS NOT NULL -- skip checking a huge list and use a null check
```
Comment (pnorman) [inline project.mml]: Can you remove the comment on the 2nd line, since the one above covers both

**Comment by kocio-pl** [comment]
Comments cleaned.

BTW: we already use simplified amenities checking for text layers and that was also what made me think that it should be done here.

**Pull request merged**